tic-tac-toe/src/components/board/board.js

```diff
@@ -0,0 +1,30 @@
+import React, {Component} from 'react';
+import Square from '../square/square';
+import './board.css';
+
+class Board extends Component {
+    renderSquare(i) {
+        return (
+            <Square
+                value={this.props.squares[i]}
+                onClick={() => {this.props.onClick(i)}}
+                key={i}
+            />
+        );
+    }   
+    render() {
+        const outerClassName = this.props.outerClassName;
+        const innerClassName = "board";
+        const className = `${outerClassName} ${innerClassName}`;
```

Comment:
It's overkill here. Just add a className to your component.

Reply:
I change this.
```suggestion
  return (
    <div className={classes.Board}>
      {squareComponents}
    </div> 
  )
```